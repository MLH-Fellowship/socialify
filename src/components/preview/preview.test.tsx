import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Preview from './preview'

beforeAll(() => {
  window.HTMLCanvasElement.prototype.toDataURL = () => ''
})

test('Preview renders', () => {
  const preview = shallow(<Preview />)

  expect(toJson(preview)).toMatchSnapshot()
})
