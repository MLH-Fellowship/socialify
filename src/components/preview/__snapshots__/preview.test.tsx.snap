// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview renders 1`] = `
<section
  className="preview-wrapper"
>
  <div
    className="preview-card-wrapper"
    onClick={[Function]}
  >
    <link
      href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;500&display=swap"
      rel="stylesheet"
    />
    <link
      href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.6/devicon.min.css"
      rel="stylesheet"
    />
    <Card
      fileType="PNG"
      font="Inter"
      logo=""
      name=""
      pattern="Plus"
      theme="Dark"
    />
    <img
      alt="Card"
      className="preview-image-wrapper"
      src="/png"
    />
  </div>
  <div
    className="preview-download-wrapper"
  >
    <Space>
      <Button
        block={false}
        className="preview-download-button"
        download="undefined-undefined.png"
        ghost={false}
        href="/png"
        htmlType="button"
        icon={<ForwardRef(DownloadOutlined) />}
        loading={false}
        type="primary"
      >
        Download
      </Button>
      <Button
        block={false}
        className="preview-copy-url-button"
        ghost={false}
        htmlType="button"
        icon={<ForwardRef(CopyOutlined) />}
        loading={false}
        onClick={[Function]}
        type="default"
      >
        Image url
      </Button>
      <Button
        block={false}
        className="preview-copy-markdown-button"
        ghost={false}
        htmlType="button"
        icon={<ForwardRef(CopyOutlined) />}
        loading={false}
        onClick={[Function]}
        type="default"
      >
        Markdown
      </Button>
      <Button
        block={false}
        className="preview-copy-tag-button"
        ghost={false}
        htmlType="button"
        icon={<ForwardRef(CopyOutlined) />}
        loading={false}
        onClick={[Function]}
        type="default"
      >
        Meta tag
      </Button>
    </Space>
  </div>
</section>
`;
